app/models/appellation.rb

```diff
@@ -0,0 +1,6 @@
+class Appellation < ApplicationRecord
+  validates_presence_of :name
+  belongs_to :wine_region
+  belongs_to :state
```

Comment:
needs optional: true on both state and wine_region. US doesn't differentiate between wine regions and appellations, and AVA is closer to an appellation than a wine region.